From 234b18e9759768a0cde2e385cdc8b076d52e5965 Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Wed, 1 Jun 2011 18:12:00 +0800
Subject: [PATCH 994/998] ENGR00143611-2  DA9053: track DA9053 version/USB wakeup issue

fix the DA9053 chip version identification

Signed-off-by: Wayne Zou <b36644@freescale.com>
(cherry picked from commit e8886aed93e71c3bf9e5f38ed87bcff948cf3d9a)
---
 drivers/mfd/da9052-core.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mfd/da9052-core.c b/drivers/mfd/da9052-core.c
index 4201922..571006b 100644
--- a/drivers/mfd/da9052-core.c
+++ b/drivers/mfd/da9052-core.c
@@ -504,7 +504,7 @@ int da9052_ssc_init(struct da9052 *da9052)
 	if ((ssc_msg.data & DA9052_CHIPID_MRC) == 0x80) {
 		da9052->chip_version = DA9053_VERSION_AA;
 		pr_info("AA version probed\n");
-	} else if ((ssc_msg.data & DA9052_CHIPID_MRC) == 0xf0) {
+	} else if ((ssc_msg.data & DA9052_CHIPID_MRC) == 0xa0) {
 		da9052->chip_version = DA9053_VERSION_BB;
 		pr_info("BB version probed\n");
 	} else {
-- 
1.7.4.1

